### DISCUSSION TITLE
Checkbox menu for individual groups in Batch Convert - "Multiple Replace".

### DISCUSSION THREAD

**Discussion by OtaStrom** [SubtitleEdit/subtitleedit#6928] [Ideas]
It would be useful to display a check box menu on the right-hand side in Batch Convert's "Multiple replace" option (similar to the "Convert colors to dialog" option), which would include the individual Groups specified in the Multiple replace settings.
This feature would be useful for differentiated corrections specifically written for a particular use (e.g. for errors in a specific VoD service).

**Comment by niksedk** [comment]
Could you make a few screenshots?

**Comment by OtaStrom** [comment]
The idea is that these groups:

<img width="173" alt="Groups" src="https://github.com/SubtitleEdit/subtitleedit/assets/79414788/36eb7498-f063-4ad5-adc5-a3c40842d16d">



would appear in the Batch convert:

<img width="449" alt="Batch convert" src="https://github.com/SubtitleEdit/subtitleedit/assets/79414788/90a3b18b-d159-4e27-972e-b50066eead6d">



Like this:

<img width="449" alt="Batch convert NEW" src="https://github.com/SubtitleEdit/subtitleedit/assets/79414788/bb5810e7-4d63-486c-ac89-e2c1e4cd2b57">
